### PR TITLE
Added Writers/UX Badges to contributing.md

### LINKED ISSUE #3532: Add the writers program UX badge to the contributing.md file
**Current Behavior**
Following up on issue #3527, it would be great to add this design to the contributing.md file like the other badges.

**Desired Situation**
Add the [writers badge](https://www.figma.com/file/SMP3zxOjZztdOLtgN4dS2W/Meshery-UI?node-id=1690%3A10102&t=BDGY3h6mSVmZiBAn-1) to the [contributing.md file](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)

---
[Optional] **Alternatives**
Screenshot 2022-12-20 at 8 54 28 PM

---
**Contributor Resources**

The layer5.io website uses Gatsby, React, and GitHub Pages. Site content is found under the [`master` branch](https://github.com/layer5io/layer5/tree/master).
- See [contributing instructions](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md)
- See Layer5 site designs in this [Figma project](https://www.figma.com/file/5ZwEkSJwUPitURD59YHMEN/Layer5-Designs). Join the [Layer5 Community](http://slack.layer5.io) for access.

### PR DESCRIPTION
**Pull request by b0id**
**Description**

Updated `contributing.md` to reflect writers/UX badges.

This PR fixes #3532

**Notes for Reviewers**

This worked and looked good on my machine.  

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

**Comment by l5io** [comment]
🚀 Preview for commit da1030eb12a1553afb8468e502a400905db1f3e7 at: https://63bf8b17944b5c047c69a53a--layer5.netlify.app

**Comment by b0id** [comment]
So I had some Git confusion.  @Nikhil-Ladha This PR affects two separate issues. Obviously I am still learning Git flow.  
I think technically it is sorted in my branch.  
See #3457 
Also #3532

**Comment by b0id** [comment]
@leecalcote I am sure you can put the right person on this to check my work.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -183,6 +183,8 @@ Badges are a great way of highlighting the area of contribution by any given com
 - <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/service-mesh-performance/stacked/smp-dark-text.svg" width="25px" height="25px"/> SMP
 - <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/nighthawk/icon-only/SVG/nighthawk-logo.svg" width="25px" height="25px"/> Nighthawk
 - <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/service-mesh-patterns/service-mesh-pattern.svg" width="25px" height="25px"/> Patterns
+- <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/ux-design/ux-design.svg" width="25px" height="25px"/> UX Design
+- <img src="https://github.com/layer5io/layer5/blob/master/src/assets/images/writer-program/writers-program.svg" width="25px" height="25px"/> Writer's program
```
Comment (leecalcote) [inline CONTRIBUTING.md:187]: Good deal. Thank you for including the extra whitespace, too.

**Comment by l5io** [comment]
🚀 Preview for commit f557b60638b87f4d01c465bbd3c6c3be3f9cdbe0 at: https://63bfa316423d1e17cf9f37a6--layer5.netlify.app

File: src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js [hunk 2]
```diff
@@ -49,7 +49,7 @@ export const ProgramsPageWrapper = styled.div`
         padding: 5px;
         color: ${props => props.theme.text};
         margin: auto;
-        background: ${props => props.theme.DarkTheme ? "rgba(10, 10, 10, 1)" : "rgba(247, 247, 247, 1)"};
+        background: ${props => props.theme.DarkTheme ? "rgba(152, 152, 152, 1)" : "rgba(247, 247, 247, 1)"};
```
Comment (Nikhil-Ladha) [inline src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js:52]: This is not the solution we intend to have.
Use different versions of MLH and the Layer5 logo for the light/dark theme.
Layer5 logo could be found under the assets directory, and the MLH logo could be downloaded from the issue, renamed, and added to the repo.

**Review by Nikhil-Ladha — CHANGES_REQUESTED** [review]
The writer-program svgs have already been added, no need to add them again.
Rebase your branch with the latest code from upstream.

File: src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js (hunk 2 above)
Comment (b0id) [reply src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js:52]: Got it, @leecalcote also mentioned background was too light, have, changed, but will change the logos as well.  Thanks @Nikhil-Ladha

**Comment by leecalcote** [comment]
A quick heads-up in case folks haven't seen this dated PR - https://github.com/layer5io/layer5/pull/3389

**Comment by Nikhil-Ladha** [comment]
The other PR adds the badge to the Members page dropdown.
However, seems like the issue that the PR was fixing was closed and stale and unfortunately leading to another PR #3607 for a similar new issue created later on 😕

File: src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js (hunk 2 above)
Comment (Nikhil-Ladha) [reply src/sections/Careers/Careers-Programs-grid/ProgramGrid.style.js:52]: I think he meant for the light mode. Not for the dark mode.

**Pull request closed without merging**